src/shared/setup.js

```diff
@@ -47,6 +48,12 @@ async function connectDatabase () {
   await Db.connect(Config.DATABASE)
   const dbLoadedTables = Db._tables ? Db._tables.length : -1
   Logger.isDebugEnabled && Logger.debug(`DB.connect loaded '${dbLoadedTables}' tables!`)
+
+  // Connect central-ledger database as well
```

Comment:
why is this needed?

Reply:
central-settlement uses models from central-ledger and those models need to have knex initialized. Otherwise, CL model calls will fail.

Reply:
Ok, maybe put that detail in the comment.

Reply:
Added